exercises/to_reject_or_not_to_reject/problem.md

```diff
@@ -34,7 +34,7 @@ function (user, callback) {
 Let's build a simple script to **prove** to ourselves that promises may only
 resolve one time and all future attempts to resolve them will simply be ignored.
```

Comment:
The instructions for this task use the word "resolve" everywhere. Unless "resolve" and "fulfill" are explicitly stated to be equivalent, one of them should be picked.